### DISCUSSION TITLE
Is it possible to run the log view with full commit messages (i.e. not `--oneline`)?

### DISCUSSION THREAD

**Discussion by rmacklin** [NeogitOrg/neogit#1279] [Q&A]
I know you can jump into an individual commit to view that specific commit's full message (along with its diff), but I'd like the ability to have a scrollable log view that (for all commits) shows the full commit messages (i.e. the summary line _and_ the body, or at least the first N lines of the body) where we could still easily navigate to the next/previous commit and jump into one to see its diff. I didn't see a way to do this, but maybe I just missed it - please let me know if it's possible.

**Comment by CKolkey** [comment]
Log view - `l l`

Tab will expand/collapse individual commits, and `c-k` `c-j` will navigate up and down within the list, opening and closing as needed.

On the nightly branch, the folded content is a real vim fold, and you can operate on it using regular fold commands.

**Reply by rmacklin** [reply]
Gotcha, thanks!

I haven't tried the nightly branch yet but this is what I'm seeing on `master`:

![image](https://github.com/NeogitOrg/neogit/assets/1863540/398a0c9b-96b3-47cd-b665-be28166e5cfb)

Some feedback: As seen in the expanded commits there, it doesn't seem to render newlines in this view, though they are rendered correctly in the commit detail view:

![image](https://github.com/NeogitOrg/neogit/assets/1863540/32eb1f0b-1c45-4d36-b79a-476f81ccbfcc)

![image](https://github.com/NeogitOrg/neogit/assets/1863540/a9b55ae3-761b-4360-acc5-fe7848baf059)

This makes it difficult as a replacement for regular `git log`. Additionally, I'm less interested in the `Author`/`AuthorDate` and especially `Commit`/`CommitDate` information when expanding these given that the `Author`/`AuthorDate` information is already presented at the end of the first line and that I can pop into the detail view if I really want to see the `Commit`/`CommitDate` information. IMO it would be ideal if we could configure the [`--pretty=` flag](https://git-scm.com/docs/git-log#_pretty_formats) as one of the Formatting options:

![image](https://github.com/NeogitOrg/neogit/assets/1863540/05cca3b1-0f95-4e4c-b701-b0b8242eca41)

Is that something you've considered and decided against? (Or is that another thing that's already improved in the nightly branch?)
